File: java/program63.java
import java.util.*;

class Pattern
{
    public int iRow;
    public int iCol;

    Pattern(int no1,int no2)
    {
        this.iRow = no1;
        this.iCol = no2;
    }

    public void DisplayPattern()
    {
        int i = 0;
        int j = 0;
        char ch = ' ';

        for(i = 1; i <= iRow; i++)
        {   
            for(j = 1,ch = 'a'; j <= iCol; j++,ch++)
            {
                System.out.print(ch+"\t");
            }
            System.out.println();
        }
    }
}

class program63
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of rows ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter number of columns");
        int iValue2 = sobj.nextInt();

        Pattern obj = new Pattern(iValue1,iValue2);
        obj.DisplayPattern();
    }
}

File: cpp/program63.cpp
#include<iostream>
using namespace std;

class Pattern
{
    public:
        int iRow;
        int iCol;

        Pattern(int X, int Y)
        {
            iRow = X;
            iCol = Y;
        }

        void DisplayPattern()
        {
            char ch = '\0';
            int i = 0,j = 0;

            for(i = 1; i <= iRow; i++)            
            {
                for(j = 1,ch = 'a'; j <= iCol; j++,ch++)
                {
                    cout<<ch<<"\t";
                }
                cout<<"\n";
            }
        }
};

int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    cout<<"Enter number of rows \n";
    cin>>iValue1;

    cout<<"Enter number of columns \n";
    cin>>iValue2;

    Pattern obj(iValue1,iValue2);
    obj.DisplayPattern();

    return 0;
}